cmd/nerdctl/image/image_inspect_test.go

```diff
@@ -71,16 +70,12 @@ func TestImageInspectSimpleCases(t *testing.T) {
 func TestImageInspectDifferentValidReferencesForTheSameImage(t *testing.T) {
 	nerdtest.Setup()
 
-	platform := runtime.GOOS + "/" + runtime.GOARCH
```

Comment:
Earlier version of this test was using linux/amd64, because it was hardcoding the sha.
When I rewrote the test a few weeks ago, I removed the static sha and replaced by a lookup, allowing to use whatever platform - though I did not remove the platform mechanism itself for some reason.
It is unnecesary.